File: PCCP 모의고사 1회 /python3/1.py
import re

def solution(input_string) :
    ALPHABET = list(map(chr, range(97, 123)))
    
    if len(input_string) > 2600 : 
        input_string = input_string[0:2600]
    
    answer = ''.join(filter(
        lambda alphabet : re.compile(f'{alphabet}+[^{alphabet}]+{alphabet}').search(input_string), ALPHABET)
                    )
    
    if answer == '' :
        answer = 'N';

    return answer

File: PROGRAMMERS/PCCP 모의고사 1회/javascript/1.js
function solution(input_string) {
    const ALPHABET = Array.from({ length: 26 }, (v, i) => String.fromCharCode(i + 97));
    input_string = (input_string.length>2600) ? input_string.slice(0, 2600) : input_string;

    let answer = ALPHABET.filter((alphabet)=>{
        const regexp = new RegExp(`${alphabet}+[^${alphabet}]+${alphabet}+`)
        return regexp.test(input_string)
    }).join('');

    if (answer === '') answer = 'N';

    return answer
}
